dandiapi/api/models/version.py

```diff
@@ -188,18 +188,19 @@ def citation(cls, metadata):
     def strip_metadata(cls, metadata):
         """Strip away computed fields from a metadata dict."""
         computed_fields = [
-            'name',
-            'identifier',
-            'version',
-            'id',
-            'url',
             'assetsSummary',
             'citation',
-            'doi',
             'dateCreated',
             'datePublished',
-            'publishedBy',
+            'doi',
+            'id',
+            'identifier',
             'manifestLocation',
+            'name',
+            'publishedBy',
+            'repository',
```

Comment:
note: `repository` is the field I added here as well for consistency -- I do not see why we should not strip it from version as well